src/suffer.cpp

```diff
@@ -311,8 +311,8 @@ void suffer::while_grabbed( Character &you )
     int grab_intensity = you.get_effect_int( effect_grabbed, body_part_torso );
 
     // you should have trouble breathing as you get swarmed by zombies grabbing you
-    if( grab_intensity <= 2 ) {
-        // only a chance to lose breath at low grab chance
+    if( grab_intensity == 2 ) {
```

Comment:
I think you may also want to add the following:
```
if ( grab_intensity < 2 ) {
    return;
}
```
Because, as you have it now, it won't exactly work when `grab_intensity` is 1, because if `grab_intensity` is 1, the `grab_intensity == 2` will be false, but then the `grab_intensity <= 4` will be true and it will do the suffocation part, which isn't desired behavior in this case.

Reply:
Thank you, I can't believe I missed that.